itest/assets_test.go

```diff
@@ -1237,7 +1263,15 @@ func assertPaymentHtlcAssets(t *testing.T, node *HarnessNode, payHash []byte,
 
 	t.Logf("Asset payment: %v", toProtoJSON(t, payment))
 
-	targetID := hex.EncodeToString(assetID)
+	var targetID string
+	switch {
+	case len(assetID) > 0:
+		targetID = hex.EncodeToString(assetID)
+
+	case len(groupID) > 0:
+		groupHash := sha256.Sum256(groupID)
```

Comment:
Same here, re x-coordinate. I wonder why this didn't cause things to fail? Or did this still point to the old commit in `tapd`?